mapsforge-map-android/src/main/java/org/mapsforge/map/android/graphics/AndroidResourceBitmap.java

```diff
@@ -82,6 +83,9 @@ public static void clearResourceBitmaps() {
                 if (bitmap == null) {
                     throw new IOException("BitmapFactory failed to decodeStream");
                 }
+                float[] finalSize = GraphicUtils.computeFinalImageSize(bitmap.getWidth(), bitmap.getHeight(), scaleFactor, width, height, percent);
+                if ((int) finalSize[0] != width || (int) finalSize[1] != height)
```

Comment:
The comparison should be done with bitmap's width and height.

Reply:
Whoops, you are right. Fixed.